GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -280,6 +279,11 @@ private void OnGuiLoadingScreenSelect()
             {
                 HandleCreateLoadingSceneButton();
             }
+
+            if (EditorGUI.EndChangeCheck())
+            {
+                Config.SaveConfiguration(ToolBarSelectedButton.LoadingScreen);
```

Comment:
I assume that every character that is typed in the AssetBundleUrl field is going to trigger a save?

One thing we were thinking about originally was not having to do too much file I/O as tiny/frequent modifications happen.

Reply:
That's true, the file's around 1KB. Depending on the typing speed of the user we'd probably be doing this somewhere between 2-5 times a second. If we want to lower the I/O further we can flip a flag when something changes and check that flag every X seconds. If it's true we trigger a save.